### PR TITLE
Feature/us4255 leader ends group

### PR DESCRIPTION
**Pull request by Doug-Shannon**
Ability for leader to end group and send auto generated email

### CODE REVIEW

File: CI/SQL/20160815_110130_US4255-add-group-ended-email-template.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+GO
+
+IF NOT EXISTS(SELECT * FROM [dbo].dp_Communications WHERE communication_id = 290789)
+BEGIN
+
+SET IDENTITY_INSERT [dbo].[dp_Communications] ON;
+
+INSERT INTO [dbo].dp_Communications 
+(Communication_ID,Author_User_ID,Subject   � � � � � � � � � � � � � � � �,Body �,Domain_ID,Start_Date � � � � � � � �,Communication_Status_ID,From_Contact,Reply_to_Contact,Template,Active) VALUES
+(290789 � � � � �,5 � � � � � � ,'Your existing group is no longer active','temp',1 � � � �,{ts '2016-08-11 00:00:00'},1 � � � � � � � � � � �,7 � � � � � ,7 � � � � � � � ,1 � � � ,1 � � );
```
Comment (sseissiger) [inline CI/SQL/20160815_110130_US4255-add-group-ended-email-template.sql]: This messageId is already being used in PROD

File: CI/SQL/20160811_152542_US4255-UpdateEndedReasonDesc.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+USE [MinistryPlatform]
+GO
+
+UPDATE [dbo].[Group_Ended_Reasons]
```
Comment (jpc11623) [inline CI/SQL/20160811_152542_US4255-UpdateEndedReasonDesc.sql]: Do these rows already exist in demo and prod?

File: Gateway/crds-angular/Controllers/API/GroupController.cs [hunk 3]
```diff
@@ -70,6 +70,28 @@ public IHttpActionResult PostGroup([FromBody] GroupDTO group)
             });
         }
```
Comment (jpc11623) [inline Gateway/crds-angular/Controllers/API/GroupController.cs]: Needs an xml comment as to the purpose of the endpoint

File: CI/SQL/20160811_152542_US4255-UpdateEndedReasonDesc.sql (hunk 2 above)
Comment (Doug-Shannon) [inline CI/SQL/20160811_152542_US4255-UpdateEndedReasonDesc.sql]: Yes, everything except for other.

**Comment by jpc11623** [comment]
Reviewed, looks good.

**Pull request merged**